pages/recipes/[slug].js

```diff
@@ -47,14 +43,15 @@ export async function getStaticPaths() {
 
 // STEP 2: tell next.js how to get data for each individual recipe
 export async function getStaticProps({ params }) {
+  const { slug } = params
   // go get the recipe data from sanity using groq
   // const recipe = sanity groq query using params.slug
-  const recipes = await sanityClient.fetch(
-    `*[_type == "recipe" && slug.current == "${params.slug}"]{
+  const recipe = await sanityClient.fetch(
+    `*[_type == "recipe" && slug.current == $slug][0]{
```

Comment:
Here we'll use GROQ params, and return the only doc as an object instead of an array `[0]`